tests/canonical/tests.rs

```diff
@@ -1,42 +1,54 @@
 #[cfg(test)]
 mod tests {
     use super::*;
-    use cbor_event::{de::Deserializer, Sz, StringLenSz};
+    use cbor_event::{de::Deserializer, StringLenSz, Sz};
     use serialization::Deserialize;
 
     fn deser_test_orig<T: Deserialize + Serialize>(orig: &T) {
         print_cbor_types("orig (original enc)", &orig.to_cbor_bytes());
-        let deser = T::deserialize(&mut Deserializer::from(std::io::Cursor::new(orig.to_cbor_bytes()))).unwrap();
+        let deser = T::deserialize(&mut Deserializer::from(orig.to_cbor_bytes())).unwrap();
         print_cbor_types("deser", &deser.to_cbor_bytes());
         assert_eq!(orig.to_cbor_bytes(), deser.to_cbor_bytes());
     }
 
     fn deser_test_canonical<T: Deserialize + Serialize>(orig: &T) {
         print_cbor_types("orig (canonical)", &orig.to_canonical_cbor_bytes());
-        let deser = T::deserialize(&mut Deserializer::from(std::io::Cursor::new(orig.to_canonical_cbor_bytes()))).unwrap();
+        let deser =
+            T::deserialize(&mut Deserializer::from(orig.to_canonical_cbor_bytes())).unwrap();
         print_cbor_types("deser", &deser.to_canonical_cbor_bytes());
-        assert_eq!(orig.to_canonical_cbor_bytes(), deser.to_canonical_cbor_bytes());
+        assert_eq!(
+            orig.to_canonical_cbor_bytes(),
+            deser.to_canonical_cbor_bytes()
+        );
     }
 
     #[test]
     fn struct_array() {
         let non_canonical_bytes = vec![
             cbor_tag_sz(11, Sz::Eight),
-                vec![ARR_INDEF],
-                    vec![0x09],
-                    cbor_string("jdskfjdsfkjad"),
-                    vec![0x42, 0xB4, 0xD3],
-                vec![BREAK]
-        ].into_iter().flatten().clone().collect::<Vec<u8>>();
+            vec![ARR_INDEF],
+            vec![0x09],
```

Comment:
I assume this is just the result of running cargo fmt or whatever. I hadn't ran it on the tests since they're tests and to preserve this indentation as it helps show the actual CBOR structure which can be helpful for reading/modifying it. e.g. what is nested in what